Makefile

```diff
@@ -439,6 +439,20 @@ $(BUILD_DIR)/assets/%.o: assets/%.c
 	$(CC) -c $(CFLAGS) $(MIPS_VERSION) $(OPTFLAGS) -o $@ $<
 	$(OBJCOPY) -O binary $@ $@.bin
 
+# Temporary hack to handle asset differences
+ifeq ($(VERSION),gc-eu-mq)
+$(BUILD_DIR)/assets/textures/nes_font_static/nes_font_static.o:
```

Comment:
Is including the baserom file in the spec not an option? (I don't know if this file provides symbols the source relies on. Even if it does though they could just go into undefined_syms.txt)

Also to be clear this is needed changes wrt moving towards OK right, it's not technically required to build the rom at all?

Reply:
I thought it would it be nice to quarantine to the Makefile since we'll have to update that to support multiversion assets. But I didn't think about including it in the spec and it is much simpler there, so I've changed to that

Reply:
> Also to be clear this is needed changes wrt moving towards OK right, it's not technically required to build the rom at all?

Right (although this did inadvertently fix the build for gc-eu-mq after the extracted/ change)

Reply:
yeah you want to basically revert #1911's Makefile changes lol, this is addressing it properly